dockerfiles/web/Dockerfile

```diff
@@ -23,7 +23,7 @@ FROM ${GRAND_CHALLENGE_WEB_BASE_REPOSITORY_URI}:${PYTHON_VERSION}-${POETRY_HASH}
 WORKDIR /app
 COPY --chown=django:django ./app/ /app/
 
-RUN python manage.py collectstatic --noinput && python manage.py compress
+RUN python manage.py collectstatic --noinput && python manage.py compress --force
```

Comment:
Only a small comment here as to why the `--force` was added. Always has a bit of an infra-as-code smell.

Reply:
We also force the other command with `--no-input`. `--force` here skips the `COMPRESS_ENABLED` check, otherwise you need to set a default to True, then override it in 3+ other places for development!